### ISSUE TITLE
Folder content disappears when adding a step

### ISSUE THREAD

**Issue by SteveRWood** [vsls-contrib/codetour#62]
CodeTour 0.0.23

![image](https://user-images.githubusercontent.com/33896295/80573872-b896b280-89f8-11ea-95dc-18b97f49deaf.png)

To demonstrate this I've created a minimal project, new class library:
![image](https://user-images.githubusercontent.com/33896295/80574092-19be8600-89f9-11ea-9568-697286715e4d.png)

I create a new tour, add a step and the folder view becomes unavailable until I remove the .tours folder and reload VS Code
![image](https://user-images.githubusercontent.com/33896295/80574333-8cc7fc80-89f9-11ea-907a-f86ca5864deb.png)

**Comment by lostintangent** [comment]
Weird! So the files are still on disk, it’s just that they aren’t being rendered in the explorer tree? If you reloaded VS Code, but didn’t delete the .tours folder, does this work? I’m just curious if the presence of the .tours folder is causing this issue, or if there’s some tree rendering problem after initially adding a new step.

I’ll look into this right now! Apologies for the inconvenience.

**Comment by lostintangent** [comment]
@gsmile Are you able to repro this? I just tried it on both a Windows and macOS machine, and I'm not seeing this issue. I also tried VS Code stable and insiders, and both seemed to work for me.

@SteveRWood If you open up the developer console (by going to the `Help > Toggle Developer Tools` menu), do you see any errors that seem related to CodeTour?

**Comment by gsmile** [comment]
@lostintangent This is strange. I'm not able to repro this on a Windows or macOS.

**Comment by SteveRWood** [comment]
Mmm. This is odd.

There is an error shown in Developer Tools:
> `No editors associated with document: /commentinput-95c0d05c-a2b2-448e-9cf6-883d5507f879-codetour.4.md`

For reference I do have tools installed for Markdown as I use VS Code as my main Markdown editor.

I carried out the following test:

- disabled the CodeTour extension
- restart VS Code
- open Class.cs
- enable the CodeTour extension
- restart VS Code
- start the recorded tour
_At this point the tour version of Class.cs opens and explorer stops showing the content of the folder and the above mentioned error is displayed in Developer Tools._
- select the other Class.cs tab and Explorer displays the full content of the folder as expected.

I do have a number of extensions installed; I will look at disabling all but CodeTour and it's dependencies incase of conflicts and will report back.

**Comment by lostintangent** [comment]
@SteveRWood Hey! I shipped an extension update that I think might have fixed this. I still haven’t been able to repro it, but I made some changes in this area, so it’s worth a shot 😁 Let me know if it’s still reproing for you. Thanks!

**Comment by SteveRWood** [comment]
@lostintangent I've now got 0.0.25 installed. I removed the old `.tour` folder and started a new tour to test.

I've managed to record the outcome; unfortunately it still is hiding the content of a folder that contains a step.
![CodeTourBug](https://user-images.githubusercontent.com/33896295/80943457-279c4e80-8ddf-11ea-870f-8611c6135d3e.gif)

I've also tested this on insiders 1.45.0 (user and system setup) with same result.

**Comment by SteveRWood** [comment]
__Update__
Not found the root of the problem; however, I have discovered that if I use refresh explorer once the content disappears, the content does re-appear until another step is created.

In my previous sample, if I add a step to `testfolder/class2.cs` the folder contents disappears. I refresh the view and the content is re-displayed. I then add a step to `testfolder/class3.cs` and the folder content disappears once more.

Here is some of the output from developer tools

![image](https://user-images.githubusercontent.com/33896295/80945300-6cc27f80-8de3-11ea-877a-1e91873fbca5.png)

**Comment by lostintangent** [comment]
@SteveRWood Does this issue still repro for you? I apologize, but I haven’t been able to repro it and I haven’t heard this issue reported by any other user. So I’m super stumped!

Could you try upgrading to the latest version of VS Code? Based on your description, this sounds potentially like a bug in VS Code, since the files are still on disk, and the steps are being recorded. I don’t even believe that it’s possible for an extension to “hide” files from the Explorer tree, so I’m stumped how this rendering issue could be happening 🤔

**Comment by SteveRWood** [comment]
@lostintangent I'm running VS Code 1.45.1 (user setup) with CodeTour 0.0.28
If I now attempt to record a code tour I get asked for a name, and then I get a notification that I can start by clicking the plus next to the line of code. However, now I don't see any plus on any line.

I've tried running with all extensions disabled, with the exception of CodeTour, I've also tried by launching VS Code with "Run as Administrator"; still no joy.

Nothing is shown in the Developer Tools
![CodeTourBug](https://user-images.githubusercontent.com/33896295/82991076-094bfc00-9ff5-11ea-8486-2b3780c64f1a.gif)
